jobs/probe_scraper.sh

```diff
@@ -7,17 +7,25 @@ BUCKET="net-mozaws-prod-us-west-2-data-pitmo"
 CACHE_DIR="probe_cache"
 OUTPUT_DIR="probe_data"
 
+ANACONDA_PATH=/mnt/anaconda2
+ANACONDA_SCRIPT=Anaconda3-5.3.1-Linux-x86_64.sh
+
+wget --no-clobber --no-verbose -P /mnt http://repo.continuum.io/archive/$ANACONDA_SCRIPT
+rm -rf $ANACONDA_PATH
+bash /mnt/$ANACONDA_SCRIPT -b -p $ANACONDA_PATH
+rm /mnt/$ANACONDA_SCRIPT
```

Comment:
Don't we need to active the conda environment (at least by adding it to `PATH`)?  Do we know what `which python3` is below?